.github/workflows/ci.yaml

```
@@ -73,3 +73,26 @@ jobs:
           file: coverage.xml
           fail_ci_if_error: False
           verbose: True
+
+  package-tests:
+    runs-on: ubuntu-latest
+    name: "package install tests"
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Micromamba
+        uses: mamba-org/provision-with-micromamba@main
+        with:
+          environment-file: environment.yml
+          environment-name: openfe
+          extra-specs: python==3.10
+
+      - name: "Install"
+        run: python -m pip install .
+
+      - name: "Tests"
+        # Move far out of the openfe src directory
+        working-directory: ../../
+        run: |
+          pytest --pyargs openfe.tests
+          pytest --pyargs openfecli.tests
```

```suggestion
          pytest -v --pyargs openfe.tests
          pytest -v --pyargs openfecli.tests
```

I find pytest without `-v` a bit hard to read, or at least I prefer this version of the output.